config/Version.public.xcconfig

```diff
@@ -1,4 +1,4 @@
 VERSION_SHORT=17.7
 
 // Public long version example: VERSION_LONG=9.9.0.0
-VERSION_LONG=17.7.0.2
+VERSION_LONG=17.7.0.3
```

Comment:
I always get caught for a second when your PRs says "beta 4" and I see ".0.3" here until I remember that iOS version numbers are zero-based 😅 👍

Reply:
Me too!

I'll try and say "beta 0" in 17.8 and see what people say 🙊